practice-02/js/index.js

```diff
@@ -71,4 +85,24 @@ document.addEventListener('DOMContentLoaded', function () {
             filterShow.style.display = 'none';
         });
     });
+
+    // Handle click event on logo buttons
+    logoButtons.forEach(function (button) {
+        button.addEventListener('click', function () {
+            const nameProduct = button.querySelector('img').getAttribute('name');
```

Comment:
why get `name` from img, can we get it right on the click button ?

Reply:
Yes, I fixed